message-bus-kafka/docker-compose.cluster.kafka.yml

```diff
@@ -1,21 +1,21 @@
 version: '3.9'
 
 services:
-  kafka:
-    hostname: kafka-{{.Node.Hostname}}
```

Comment:
I think we need hostname to all the 3 instances. if we don't specify it Kafdrop will be linked to a random hostname and if kafka fails the hostname will be updated, kafdrop will not be able to reconnect to kafka after that